packages/core/src/interaction-output.ts

```diff
@@ -42,7 +42,9 @@ export class InteractionOutput implements WoT.InteractionOutput {
         this.form = form;
         this.schema = schema;
 
-        this.data = ProtocolHelpers.toWoTStream(content.body);
+        if (content && content.body) {
```

Comment:
`body` should be mandatory in content (and also `content` should be different from undefined), do we have a calling code that is passing undefined content?

Reply:
There were issues with it, e.g., failing tests with actions that do not return any data.

I thought this is safer since (at least at the moment the http tests fail without).
I added your point to the follow-up list